js/core.js

```diff
@@ -1144,14 +1144,13 @@ async function parsePhraseBook(s, loadSketch) {
         if (importedSketches[o.name]) {
             sketch = importedSketches[o.name];
         } else {
-            let snap = await loadSketch.download(`sketch/${o.name}`);
-            sketch = Object.assign({}, snap);
-            if (sketch.source === undefined) {
+            sketch = await loadSketch(o.name);
+            if (sketch === null) {
```

Comment:
Is this null check enough? If loading fails, the method will throw, not return null.

Reply:
So I should add `sketch === undefined` ?
Is there a way to actually make the loading to fail, just to test?
Cause it returns `null` when the `sketch` key is wrong, but I am not so sure about when the loading fails.